### ISSUE TITLE
HE conceded on non-MIB tree

### ISSUE THREAD

**Issue by Mati-Roy** [oughtinc/mosaic#758]
@zjmiller 

not important. but do you know why it says "honest conceded" here: https://mosaic.ought.org/w/7692/compactTree?expanded=true

it's non-MIB:
```
SELECT *
FROM public."Trees" T
JOIN public."Workspaces" W
ON W."rootWorkspaceId" = T."rootWorkspaceId"
WHERE W."serialId" = '7692'
```

and given the sub-tree was played, this would imply the honest expert hadn't conceded

**Comment by zjmiller** [comment]
Yeah you're right that the honest expert didn't actually concede. This bug seems to occur when viewing certain subtrees with the compact tree view (you can see it doesn't happen with the whole tree, and with other certain subtrees containing it).
